src/main/java/picard/arrays/illumina/Build37ExtendedIlluminaManifestRecord.java

```diff
@@ -40,9 +40,16 @@
         ILLUMINA_FLAGGED,   // Illumina flagged
         LIFTOVER_FAILED,
         UNSUPPORTED_GENOME_BUILD,
-        SEQUENCE_MISMATCH,      // mismatch between what the manifest claims is the reference vs. the actual reference.
-        INDEL_SEQ_MISMATCH,     // Unable to reconcile indel situation
-        INDEL_EXTENSION_ERROR,  // extension base conflict.
+        PROBE_SEQUENCE_MISMATCH,        // Probe sequence not found in reference.
+        PROBE_SEQUENCE_STRAND_INVALID,  // Probe sequence is on unexpected strand.
+        SOURCE_SEQUENCE_MISMATCH,       // Source sequence not found in reference.
+        SOURCE_SEQUENCE_INVALID,        // Source sequence is invalid (contains invalid character).
+        SOURCE_SEQUENCE_STRAND_INVALID, // Source sequence is on unexpected strand.
+        INDEL_NOT_FOUND,                // Neither Insertion nor Deletion sequence found in reference
+        INDEL_CONFLICT,                 // Both insertion and deletion sequence found in reference.
+        SEQUENCE_MISMATCH,              // Deprecated - but used in existing extended manifest files
```

Comment:
Use javadoc and `@Deprecated` to deprecate these fields, e.g.
```
/** @deprecated only used in existing manifest files */
@Deprecated
SEQUENCE_MISMATCH,
```

Reply:
Yup.